README.md

```diff
@@ -94,9 +94,22 @@ For a similar reason, this polyfill can't proxy `Array` objects very well - but
 
 # Usage
 
+## To assign Proxy to the global object:
 Include the JavaScript at the start of your page, as an ES6 module (although browsers that support ES6 modules support `Proxy` natively) or include it as a dependency to your build steps.
 The source is in ES6, but the included, minified version is ES5.
 
+## To consume Proxy as a function
+Require from your app the file `./src/proxy.js`, which exports a proxy polyfill function.
+```js
+// es6/es2015 imports
+import proxyPolyfill from 'proxy-polyfill/src/proxy';
```

Comment:
This doesn't work without a transpiler? Using commonJS is fine but don't imply that ES6 modules magically integrate :)

Reply:
Of course, transpiling is still needed in most environments. Replacing with "In ES6/ES2015 supporting environments".